lib/log-config.js

```diff
@@ -1,10 +1,21 @@
 var bunyan = require('bunyan');
+var PrettyStream;
+var stream;
+
+var loggerSettings = {
+  name: 'api-webmaker-org',
+  level: process.env.LOG_LEVEL || 'info'
+};
+
+if ( process.env.NODE_ENV !== 'production' ) {
+  PrettyStream = require('bunyan-prettystream');
```

Comment:
You can also just pipe the logs to `bunyan` instead of using a pretty printer. Much more flexible than just the formatter as you can filter by log level, etc.
https://github.com/trentm/node-bunyan#cli-usage